README.rst

```diff
@@ -79,8 +79,13 @@ Build OpenMS following the OpenMS wiki instructions. Example cmake command on wi
 STEP 3: Build SmartPeak dependencies
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Download the latest SDL2 libraries. Add the SDL2 folder to the path environmental variable. Compile using cmake and build for "external projects"
+<<<<<<< HEAD
```

Comment:
It looks like there are some unresolved merge conflicts?

Reply:
Updated.